src/main.rs

```diff
@@ -1,11 +1,15 @@
 use clap::Parser;
+use const_str::concat;
+use shadow_rs::shadow;
 use tracing::{debug, info, warn};
 
+shadow!(build);
+
 mod log;
 mod utils;
 
 #[derive(Parser)]
-#[command(version)]
+#[command(version = concat!("v",build::PKG_VERSION,"-",build::SHORT_COMMIT))]
```

Comment:
We need display hash version, usually, `concat` string and `v` prefix is unnecessary